parsec/class/lifo.h

```diff
@@ -252,6 +252,19 @@ LIFO_STATIC_INLINE void parsec_lifo_push( parsec_lifo_t* lifo,
     PARSEC_ITEM_ATTACH(lifo, item);
 
     do {
+#if 0
+        parsec_counted_pointer_t old_head;
+        old_head.data = lifo->lifo_head.data;
+        parsec_atomic_rmb ();
+
+        item->list_next = lifo->lifo_head.data.item;
+
+        if (parsec_update_counted_pointer(&lifo->lifo_head, old_head,
+                                          (parsec_list_item_t *)item)) {
+            parsec_atomic_wmb ();
+            return;
+        }
+#else
```

Comment:
Do we want to keep the code that is disabled here? A comment on what it does would be good.